SolutionInfo.cs

```diff
@@ -6,10 +6,8 @@
 [assembly: AssemblyVersionAttribute("0.19.0")]
 [assembly: AssemblyFileVersionAttribute("0.19.0")]
 [assembly: ComVisibleAttribute(false)]
-namespace System
-{
-    internal static class AssemblyVersionInformation
-    {
+namespace System {
+    internal static class AssemblyVersionInformation {
```

Comment:
This is the eternal struggle between the FAKE script and `FormatCode` - see https://github.com/shiftkey/Octokit.CodeFormatter/issues/2 for how I'd like to tackle this...